engine/consensus/dkg/reactor_engine.go

```diff
@@ -60,7 +60,7 @@ func NewReactorEngine(
 ) *ReactorEngine {
 
 	logger := log.With().
-		Str("engine", "dkg_reactor").
+		Str("engine", "dkg.reactor").
```

Comment:
Is this intentional? Usually, component names are separated by an underscore (_), as seen in previous examples.

Reply:
Fixed in https://github.com/onflow/flow-go/pull/4219/commits/56af6759e9df034a1b1c66a4649cb83320aa6967